routers/api/v1/repo/tree_test.go

```diff
@@ -5,9 +5,10 @@
 package repo
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Can you move this import below the `code.gitea.io/...` imports (with an empty line between)?